README.md

```diff
@@ -0,0 +1,10 @@
+# Gigantic Parser Module
+
+A Module to Parse Data from *Motiga's* **Gigantic**
+
+
+## Installation:
+
+```
+pip install -e .[development]
```

Comment:
This is the development command, the other command is just `pip install .` or `python setup.py develop`, if you can edit this for that clarification I'll merge it.